analyzer/src/main/kotlin/managers/Poetry.kt

```diff
@@ -58,8 +58,9 @@ class Poetry(
 
         log.info { "Generating '${requirementsFile.name}' file in '$workingDir' directory..." }
 
-        val requirements = ProcessCapture(workingDir, command(), "export", "--without-hashes", "--format=requirements.txt").requireSuccess().stdout
-        requirementsFile.writeText(requirements)
+        val req = ProcessCapture(workingDir, command(), "export", "--without-hashes", "--format=requirements.txt")
+        .requireSuccess().stdout
```

Comment:
This should be indented by four more space, I'm surprised that detekt does not complain, but maybe it's already fixed in a later commit?